### PR TITLE
Add droplet run to merge flow

### PR DESCRIPTION
**Pull request by joshuef**

### CODE REVIEW

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Review by jacderida — APPROVED** [review]

File: .github/workflows/merge.yml [hunk 1]
```diff
@@ -10,8 +10,19 @@ concurrency:
   cancel-in-progress: true
 
 env:
+  NODE_COUNT: 30
```
[the remark this answers is not in the document]
Comment (joshuef) [reply .github/workflows/merge.yml:13]: No you're right, until that's stable it shouldnt be in.

File: .github/workflows/merge.yml [hunk 2]
```diff
@@ -546,6 +562,8 @@ jobs:
     if: "!startsWith(github.event.pull_request.title, 'Automated version bump')"
     name: Run API tests
     runs-on: ${{ matrix.os }}
+    env:
+      NODE_COUNT: 15
```
[the remark this answers is not in the document]
Comment (joshuef) [reply .github/workflows/merge.yml:566]: nope. i've removed the 30

**Review by jacderida — DISMISSED** [review]

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Comment by joshuef** [comment]
bors try

**Pull request closed without merging**